src/billing/components/marketplace/MarketplaceBilling.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {FC} from 'react'
+
+import {
+  Panel,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  AlignItems,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+import MarketplaceLink from 'src/billing/components/marketplace/MarketplaceLink'
+import PurplePanel from 'src/billing/components/marketplace/PurplePanel'
```

Comment:
#nit I know you copied this straight from Quartz, but is there a better name than `PurplePanel` for this? Is it even purple any more? :-)